frontend/src/locale/src/ar.json

```diff
@@ -0,0 +1,560 @@
+{
+	"access-lists.title": {
+		"defaultMessage": "قوائم الوصول"
+	},
+	"acmesh.dns_acmedns": {
```

Comment:
It's worth noting that if a definition doesn't exist here it will fall back to the `en.json` value. For lang keys starting with `acmesh.` you don't need to add them here since they _should_ have the same value in each language.